flutter/pubspec.yaml

```diff
@@ -37,7 +37,7 @@ flutter:
       ios:
         pluginClass: SentryFlutterPlugin
       macos:
-        pluginClass: SentryFlutterPluginApple
+        pluginClass: SentryFlutterPlugin
```

Comment:
was this a bug before?

Reply:
Previously I just included the SwiftPlugin file for macOS but now I included every file which is also included in iOS. So I consolidated everything between iOS and macOS.

Reply:
nice :)